src/tests/conformance/handshake/ignore_message_inplace_of_verack.rs

```diff
@@ -275,7 +263,7 @@ mod when_node_initiates_connection {
         run_test_case(Message::NotFound(block_inv)).await.unwrap();
     }
 
-    /// Checks that `message` gets ignored when sent instead of [`Message::Verack`] when the node
+    /// Checks that `message` doesn't terminate the connection when sent instead of [`Message::Verack`], when the node
     /// initiates the connection.
```

Comment:
```suggestion
    /// Checks that `message` doesn't terminate the connection when sent instead of [`Message::Verack`], when the node
    /// initiates the connection and instead ignores the message.
```